charts/images.yaml

```diff
@@ -4,10 +4,6 @@ images:
   repository: eu.gcr.io/gardener-project/gardener/terraformer-gcp
   tag: "v2.18.1"
 
-- name: cloud-controller-manager
-  sourceRepository: github.com/kubernetes/kubernetes
-  repository: k8s.gcr.io/hyperkube
```

Comment:
You can also update

https://github.com/gardener/gardener-extension-provider-gcp/blob/64c286b1fd4567ccde670acbf56e9e83f0fd6910/example/10-fake-shoot-controlplane.yaml#L150

and consider the following diff

```diff
diff --git a/hack/api-reference/api.json b/hack/api-reference/api.json
index d50497d1..841a7a39 100644
--- a/hack/api-reference/api.json
+++ b/hack/api-reference/api.json
@@ -9,7 +9,7 @@
     "externalPackages": [
         {
             "typeMatchPrefix": "^k8s\\.io/(api|apimachinery/pkg/apis)/",
-            "docsURLTemplate": "https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#{{lower .TypeIdentifier}}-{{arrIndex .PackageSegments -1}}-{{arrIndex .PackageSegments -2}}"
+            "docsURLTemplate": "https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#{{lower .TypeIdentifier}}-{{arrIndex .PackageSegments -1}}-{{arrIndex .PackageSegments -2}}"
         }
     ],
     "typeDisplayNamePrefixOverrides": {
diff --git a/hack/api-reference/config.json b/hack/api-reference/config.json
index 33b5ff4e..bd081068 100644
--- a/hack/api-reference/config.json
+++ b/hack/api-reference/config.json
@@ -17,7 +17,7 @@
         },
         {
             "typeMatchPrefix": "^k8s\\.io/(api|apimachinery/pkg/apis)/",
-            "docsURLTemplate": "https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#{{lower .TypeIdentifier}}-{{arrIndex .PackageSegments -1}}-{{arrIndex .PackageSegments -2}}"
+            "docsURLTemplate": "https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#{{lower .TypeIdentifier}}-{{arrIndex .PackageSegments -1}}-{{arrIndex .PackageSegments -2}}"
         },
         {
             "typeMatchPrefix": "github.com/gardener/gardener/extensions/pkg/apis/config",
```